scripts/automation/trex_control_plane/interactive/trex/examples/astf/tunnel_regression.py

```diff
@@ -0,0 +1,200 @@
+#!/usr/bin/python
```

Comment:
how do u plan to run this?
and may be remove it?

Reply:
This is under examples. Anyone can see how to use it. Also, can test it after making changes.